cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -16,6 +16,10 @@ spec:
       labels:
         app: rook-ceph-operator
         chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    {{- if .Values.annotations }}
+      annotations:
+{{ toYaml .Values.annotations | indent 8 }}
+    {{- end }}
```

Comment:
One last thing here could you move the beginning of the `if` and `end` to the beginning of the lines like below.